src/argus_htmx/templates/htmx/user/settings.html

```diff
@@ -0,0 +1,21 @@
+{% extends "htmx/base.html" %}
+
+{% block main %}
+    <section id="user-settings">
+        <h2>{{ page_title }}</h2>
```

Comment:
Are we not supposed to signal somehow to non-screenreading users that they are on the user settings page, @podliashanyk ?

Reply:
I added 95df10330e03b4191faab5444fbc52fdd02b79bd to resolve most of your comments, but it also adds a card title that reads "User settings" - but I don't think this is a good solution either, especially if we may resort to adding multiple cards for settings groups.

Again, we should resolve how to signal to an able-eyed user which page they are currently on, without forcing them to rely on reading the URL in the address bar - but it doesn't seem the current incarnation of argus-htmx has any kind of page heading system (maybe because we've only been focused on implementing the incidents list thus far).

Reply:
I'm a fan of breadcrumbs myself. And "prose" will not work with cards and forms inside, will it?